agent/test/get_main_net.spec.js

```diff
@@ -23,603 +23,581 @@
  * @copyright SKALE Labs 2019-Present
```

Comment:
Maybe let's rename this file to `agentUnitTests.js`

Reply:
Will rename. Thanks.